src/brogue/Rogue.h

```diff
@@ -886,10 +844,8 @@ enum wandKind {
     WAND_DOMINATION,
     WAND_BECKONING,
     WAND_PLENTY,
-    NUMBER_GOOD_WAND_KINDS = WAND_PLENTY,
```

Comment:
It for a world where a variant has additional wands 'off-the-end' of the enum (and we allow the variants to hook into the end of useItem-style functions). Honestly, NUMBER_GOOD_WANDS etc. would need to be handled differently in that case.

Reply:
Hmm, ok, so it was removed with the expectation that it will cease to be a meaningful abstraction. I suppose if it's not currently used, that makes sense, someone can add it back if truly desired